nbdime/webapp/nbdimeserver.py

```diff
@@ -257,6 +257,8 @@ def _build_arg_parser():
     return parser
 
 
-if __name__ == "__main__":
-    arguments = _build_arg_parser().parse_args()
-    main(port=arguments.port, cwd=arguments.workdirectory)
```

Comment:
I would keep this as:
```python
if __name__ == "__main__":
    main()
```
so that the file can still be executed directly (`python nbdimeserver.py`). Is there any reason why we do not want this behavior? The `nd*-web` also pop open a browser and need a set of file names, while direct execution allows for a general server.